examples/logger.py

```diff
@@ -3,13 +3,33 @@
 """
 
 import time
+import logging
 from typing import Any, Optional, Tuple
 
 from PyQt5.QtCore import QObject, pyqtSlot, pyqtSignal
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QPushButton, QTextEdit, QLabel, QDialogButtonBox
 
 from qiwis import BaseApp
 
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+class LoggingHandler(QObject, logging.Handler):
+    """
+    Handler for logger, handle input log by emit
+    """
```

Comment:
We don't break lines for a single line docstring, for example:
`"""Handler for logger, handle input log by emit."""`

In addition, please finish your sentence with a period(.).

Reply:
However, since we write a docstring for public signals, you should write it for `signal`.
For example:
```python
class LoggingHandler(...):
    """Handler for logger ...

    Signals:
        signal(log): A formatted log message is emitted.
    """
```

You can check other example docstrings in qiwis.py as well (e.g., `Qiwis`).